src/pages/Governance/Proposal/Header.tsx

```diff
@@ -47,6 +50,10 @@ function TimeRemainingComponent({
   proposal: Proposal;
   isOnMobile: boolean;
 }) {
+  if (isVotingClosed(proposal)) {
```

Comment:
not sure if we dont want to show anything... maybe we can display the time it was closed?

Reply:
I thought about it but with the current design there isn't a good way to display the time it was closed so I just hide it for simplicity. Let me add a TODO here and revisit after consulting Josh. Btw do you know what is the accurate "close time"?